pkg/orchestrator/provider/aws/scanestimation/estimate.go

```diff
@@ -304,15 +371,8 @@ func findMatchingStatsForInputTypeRootFS(stats *[]models.AssetScanInputScanStats
 	return models.AssetScanInputScanStats{}, false
 }
 
-const constantOfProportionality = 2.5
-
 func calculateStaticScanDuration(familyType familiestypes.FamilyType, scanSizeGB float64) int64 {
-	if scanSizeGB < 1 {
-		// The log of a value less than 1 is negative.
-		return int64(familyScanDurationPerGBMap[familyType].Seconds() * scanSizeGB)
-	}
-
-	return int64(familyScanDurationPerGBMap[familiestypes.SBOM].Seconds() * constantOfProportionality * math.Log(scanSizeGB))
+	return int64(familyCoefficients[familyType][0]*scanSizeGB + familyCoefficients[familyType][1])
```

Comment:
Shouldn't we be taking the natural log of scanSizeGB here?

Reply:
Good catch thank you

Reply:
Right, according to the example:
```
	xToExtrapolate := 22.5
	lnXToExtrapolate := math.Log(xToExtrapolate)
	y := a*lnXToExtrapolate + b

	fmt.Printf("Y=%.2f\n", y)
```